implementations/rust/daemon/src/cli.rs

```diff
@@ -0,0 +1,366 @@
+use std::convert::TryInto;
+use std::net::SocketAddr;
+use std::path::PathBuf;
+use std::str::FromStr;
+
+use ockam_message::message::{Route, RouterAddress};
+
+use structopt::{clap::ArgSettings::Hidden, StructOpt};
+use url::Url;
+
+/// The port on which the config updater runs and accepts Config messages.
+pub const DEFAULT_CONFIG_PORT: u16 = 11199;
+
+const DEFAULT_LOCAL_SOCKET: &str = "127.0.0.1:11200";
```

Comment:
Are the ports chosen randomly?

Reply:
That *should* work, I haven't tested it though.

Reply:
What I mean is where did `11199` and `11200` come from? Did you randomly pick them? Sorry for not being clear earlier.

Reply:
haha oh yes, I randomly picked those.. 111 = ASCII `o`, 99 = ASCII `c` (i.e. **oc**kam). I changed the code to use a any available port though, but it can still be configured at the CLI.